src/Message/AbstractOrderRequest.php

```diff
@@ -156,14 +156,17 @@ public function setWidgetOptions($widgetOptions)
     protected function getOrderData()
     {
         $data = [
-            'locale' => str_replace('_', '-', $this->getLocale()),
             'order_amount' => $this->getAmountInteger(),
             'order_tax_amount' => $this->toCurrencyMinorUnits($this->getTaxAmount()),
             'order_lines' => $this->getItemData($this->getItems()),
-            'purchase_country' => explode('_', $this->getLocale())[1],
             'purchase_currency' => $this->getCurrency(),
         ];
 
+        if (null !== $locale = $this->getLocale()) {
```

Comment:
you should add a test path for the situation when this is not null

Reply:
The test path for this situation were already there (it was mandatory until this moment). I've altered an existing test instead to check the situation when a locale has been omitted.